lib/kafka/gzip_codec.rb

```diff
@@ -0,0 +1,28 @@
+module Kafka
+  class GzipCodec
```

Comment:
it;s not a shared behaviour - it is a dsl that would allow people to build things on top of this library